LibertyDSNP/example-client
# package updates made

Purpose
---------------
Chore
Lots of our packages are out of date. Update the ones that are easy and make chores for the others.
[https://www.pivotaltracker.com/story/show/179607219](https://www.pivotaltracker.com/story/show/179607219)


Solution
---------------
- run `npm outdated`
- run `npm upddate`
- debug error messages


Change summary
---------------
* updated package lock
* updated package


Steps to Verify
----------------
1. run the app and make sure that it runs as expected
2. check out changes


Additional details / screenshot
----------------
- Could not figure out how to remove postcss
